static/js/localstorage.js

```diff
@@ -63,10 +63,14 @@ const ls = {
         localStorage.removeItem(key);
     },
 
-    // Remove keys which match a regex.
-    removeDataRegex(version, regex) {
+    // Remove keys which (1) map to a value that satisfies a
+    // property tested by `condition` and (2) which match the
+    // pattern given by `name`.
+    removeDataRegexWithCondition(version, regex, condition) {
```

Comment:
I'm not that happy with `condition` and open to other names for this variable

Reply:
Maybe something like `condition_checker`? Idunno.